mne/preprocessing/tests/test_peak_finder.py

```diff
@@ -1,10 +1,40 @@
-from numpy.testing import assert_array_equal
-
+from numpy.testing import assert_array_equal, assert_raises, assert_equal
+import numpy as np
+from mne.utils import run_tests_if_main
 from mne.preprocessing.peak_finder import peak_finder
 
 
 def test_peak_finder():
     """Test the peak detection method"""
-    x = [0, 2, 5, 0, 6, -1]
-    peak_inds, peak_mags = peak_finder(x)
+
+    # check for random data
+    peak_inds, peak_mags = peak_finder(np.random.random(20))
```

Comment:
we tend to fix the seeds to avoid random bugs. So it should be:

```
rng = np.random.RandomState(42)
peak_inds, peak_mags = peak_finder(rng.random(20))
```

Reply:
The proposed command yields an error (rng does not have a module called random). So I just defined the random seed an kept the random number creation untouched.

Reply:
you missed the other half :)

Reply:
try rng.randn

Reply:
:P - I know that randn works - but is it MNE-ish to create the rng object, because the other way you'd save an object

Reply:
Didn't feel like I would be in the right position to complain :D - so I changed it to the rng...